pages/index.tsx

```diff
@@ -48,44 +51,50 @@ const Blurb = styled.div`
 `;
 
 export default function Index() {
-  const [tabIndex, setTabIndex] = useState(0);
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
-  const [selectedIndex, setSelectedIndex] = useState(0);
+  const [selectedMenuItem, setSelectedMenuItem] = useState<string>(
+    "General Info"
+  );
   const { address: collAddress } = Collateral.useContainer();
   const { address: tokenAddress } = Token.useContainer();
   const { contract: weth } = WethContract.useContainer();
+  const { empAddress } = EmpAddress.useContainer();
 
   const isYieldToken =
     tokenAddress &&
     Object.keys(YIELD_TOKENS).includes(tokenAddress.toLowerCase());
 
-  const options = ["General Info", "Manage Position", "All Positions"];
-
-  if (isYieldToken) {
-    options.push("yUSD Yield");
-  }
+  let options = ["General Info", "Manage Position", "All Positions"];
+  const buildOptionsList = () => {
+    // If it is weth collateral contract then add the weth option.
+    if (weth && collAddress?.toLowerCase() == weth.address.toLowerCase()) {
+      options.push("Wrap/Unwrap WETH");
+    }
+
+    // If it is a yield token then add the yUSD yield and analytics tabs.
+    if (isYieldToken) {
+      options = options.concat(["yUSD Yield", "Analytics"]);
```

Comment:
What's the definition of a yield token? Just a USD tracking one? Does the analytics tab still make sense on a non-usd tracking token that has liquidity mining? What about USD-tracking tokens that have no liquidity mining?

Reply:
Its any token found in this [dictionary](https://github.com/UMAprotocol/emp-tools/blob/master/constants/yieldTokens.ts#L11) which is used to determine when to render the Yield calculator tab for example